[BUG] `show_versions` error

### Describe the bug

`from aeon import show_versions; show_versions()` raises an error when called.

### Steps/Code to reproduce the bug

```python
from aeon import show_versions; show_versions()
```

### Expected results

No error is thrown and versions are printed.

### Actual results

```python-traceback
Traceback (most recent call last):
  File "D:\CMP Machine Learning\Repositories\aeon-tsml\local_code.py", line 1, in <module>
    from aeon import show_versions; show_versions()
  File "D:\CMP Machine Learning\Repositories\aeon-tsml\aeon\utils\_maint\_show_versions.py", line 85, in show_versions
    deps_info = _get_deps_info()
  File "D:\CMP Machine Learning\Repositories\aeon-tsml\aeon\utils\_maint\_show_versions.py", line 73, in _get_deps_info
    mod = importlib.import_module(modname)
  File "C:\Users\Matthew Middlehurst\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "D:\CMP Machine Learning\Repositories\aeon-tsml\venv\lib\site-packages\setuptools\__init__.py", line 8, in <module>
    import _distutils_hack.override  # noqa: F401
  File "D:\CMP Machine Learning\Repositories\aeon-tsml\venv\lib\site-packages\_distutils_hack\override.py", line 1, in <module>
    __import__('_distutils_hack').do_override()
  File "D:\CMP Machine Learning\Repositories\aeon-tsml\venv\lib\site-packages\_distutils_hack\__init__.py", line 77, in do_override
    ensure_local_distutils()
  File "D:\CMP Machine Learning\Repositories\aeon-tsml\venv\lib\site-packages\_distutils_hack\__init__.py", line 64, in ensure_local_distutils
    assert '_distutils' in core.__file__, core.__file__
AssertionError: C:\Users\Matthew Middlehurst\AppData\Local\Programs\Python\Python310\lib\distutils\core.py
```

### Versions

I can't!

--- a/aeon/utils/_maint/_show_versions.py
+++ b/aeon/utils/_maint/_show_versions.py
@@ -10,9 +10,9 @@ adapted from :func:`sklearn.show_versions`
 __author__ = ["mloning"]
 __all__ = ["show_versions"]
 
-import importlib
 import platform
 import sys
+from importlib.metadata import PackageNotFoundError, version
 
 
 def _get_sys_info():
@@ -67,13 +67,8 @@ def _get_deps_info():
 
     for modname in deps:
         try:
-            if modname in sys.modules:
-                mod = sys.modules[modname]
-            else:
-                mod = importlib.import_module(modname)
-            ver = get_version(mod)
-            deps_info[modname] = ver
-        except ImportError:
+            deps_info[modname] = version(modname)
+        except PackageNotFoundError:
             deps_info[modname] = None
 
     return deps_info

